monthly-exams/exam-1/src/App.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import './App.css';
+
+function App() {
+  let rainbow = ['red', 'green', 'blue', 'purple', 'orange', 'black'];
+  const [currentColorIndex, setColorIndex] = useState();
+  const currentColor = rainbow[currentColorIndex];
+
+  let newIndex;
+  const onClickButton = () => {};
+
+  return (
+    <div className={`container ${currentColor}`} onClick={onClickButton}>
```

Comment:
This is not correct to add `onClick` on the `<div>`, the elements you are interacting with are the `<button>` below so `onClick` has to be set on them instead